test/server/cards/02_SHD/units/CartelTurncoat.spec.ts

```diff
@@ -0,0 +1,51 @@
+describe('Cartel Turncoat', function() {
+    integration(function(contextRef) {
+        describe('Cartel Turncoat\'s Bounty ability', function() {
+            it('should draw a card', function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        spaceArena: ['cartel-turncoat']
+                    },
+                    player2: {
+                        spaceArena: ['razor-crest#reliable-gunship']
+                    }
+                });
+
+                const { context } = contextRef;
+
+                context.player1.clickCard(context.cartelTurncoat);
+                context.player1.clickCard(context.razorCrest);
+                expect(context.player2).toHaveExactPromptButtons(['Bounty: Draw a card', 'Pass']);
```

Comment:
Can just use 'toHavePassAbilityPrompt' I think.

Same comment for most of these bounty tests